client/core/bookie.go

```diff
@@ -511,6 +511,8 @@ func handleTradeSuspensionMsg(c *Core, dc *dexConnection, msg *msgjson.Message)
 			tracker.metaData.Host == dc.acct.host && tracker.metaData.Status == order.OrderStatusBooked {
 			// Locally revoke the purged book order.
 			tracker.revoke()
+			details := fmt.Sprintf("Order %v revoked.", tracker.Order.ID())
+			c.notify(newOrderNote("Order Revoked", details, db.WarningLevel, tracker.coreOrderInternal()))
```

Comment:
Unique subjects when possible. How about `Order Auto-revoked` here, and leave `Order Revoked` below.